include/dmp_dv.h

```diff
@@ -239,6 +239,12 @@ int64_t dmp_dv_cmdlist_exec(dmp_dv_cmdlist cmdlist);
 int dmp_dv_cmdlist_wait(dmp_dv_cmdlist cmdlist, int64_t exec_id);
 
 
+/// @brief Get the last execution time in microseconds of specified command.
+/// @param cmdlist Handle to command list, when NULL the error is returned.
+/// @return last execution time in microseconds(us), or 0 if error.
+uint64_t dmp_dv_cmdlist_get_last_exec_time(dmp_dv_cmdlist cmdlist);
```

Comment:
Change return type to `int64_t` and comment to
`/// @return Last execution time in microseconds (us), or -1 on error.`

Reply:
How about remain return in `uint64_t` type, but return `(uint64_t)-1` if error?

Reply:
Better to change to `int64_t` as the range of `int64_t` is enough for microseconds and valid elapsed time `>= 0`.

Reply:
Fixed. Kernel driver interface is also updated to use s64 type as returned execution time also.